app/views/admin/articles/new.html.erb

```diff
@@ -34,6 +34,7 @@
           <%= render partial: "input_item" %>
           <%= render partial: "input_instagram" %>
           <%= render partial: "input_image" %>
+          <%= render partial: "input_place" %>
```

Comment:
__[rails_best_practices]__
* simplify render in views [:ref](http://rails-bestpractices.com/posts/2010/12/04/simplify-render-in-views/)